templates/properties.yml

```diff
@@ -217,15 +217,15 @@ meta:
       #####################
       # DIRECTOR SETTINGS #
       #####################
-      #bootstrap_bosh_director settings
+      #bosh-sf_director settings
```

Comment:
as a developer, do I need to give settings for bosh-sf for sf-boshrelease deployment to work in boshlite?

Reply:
Not really. We have planned to remove it. Will remove it.